doc/en/usage.rst

```diff
@@ -522,9 +522,7 @@ Will result in:
 
 .. warning::
 
-    ``record_property`` is an experimental feature and may change in the future.
-
-    Also please note that using this feature will break any schema verification.
+    Please note that using this feature will break schema verifications for the latest JUnitXML schema.
```

Comment:
Should `record_property` be deprecated and later removed because of this?

Reply:
Good point, but I don't think so, because it might still be useful for users of the `xunit1` standard or even custom parsers.

Reply:
I'd suggest a `PendingDeprecationWarning`, which should explain *why* the change is happening, because plugin authors who would want to know (i.e., run with `-W error`) might not have any other way to find out.

Reply:
Not sure if we want to deprecate this at all, for the reasons I mentioned before: this is still useful for people using custom parsers (which I believe was the motivation for the contribution in the first place).